apps/user-profile/src/app/pages/settings-page/settings-authorization/settings-authentication.component.ts

```diff
@@ -47,7 +47,7 @@ export class SettingsAuthenticationComponent implements OnInit {
     translate.get('AUTHENTICATION.DELETE_IMG_DIALOG_DESC').subscribe(res => this.removeDialogDescription = res);
   }
 
-  @ViewChild(MatPaginator) paginator: MatPaginator;
+  @ViewChild(TableWrapperComponent, {static: true}) child: TableWrapperComponent;
```

Comment:
![image](https://user-images.githubusercontent.com/32871167/122243241-40bd6d00-cec4-11eb-8078-20946c77749d.png)
On user profile happens this. No idea if it is caused by your PR